sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -1075,9 +1079,8 @@ internal async Task<DateTimeOffset> RenewSessionLockInternal(TimeSpan timeout)
 
             if (!_receiveLink.TryGetOpenedObject(out ReceivingAmqpLink receiveLink))
             {
-                receiveLink = await _receiveLink.GetOrCreateAsync(UseMinimum(_connectionScope.SessionTimeout, timeout)).ConfigureAwait(false);
```

Comment:
We should not attempt to reopen a closed session link as this can cause an inconsistent error experience, i.e. you would get an AcceptSession error rather than SessionLockLost.